marklogic-data-hub-spark-connector/src/main/java/com/marklogic/hub/spark/sql/sources/v2/reader/HubDataSourceReader.java

```diff
@@ -91,7 +91,7 @@ protected ObjectNode buildInitializeReadInputs(Map<String, String> options) {
 
         // The same all-lowercase style is used here for consistency with Spark, even though it's not consistent with
         // DHF code conventions
-        Stream.of("view", "schema", "sqlcondition", "selectedcolumns").forEach(key -> inputs.put(key, options.get(key)));
+        Stream.of("view", "schema", "sqlcondition", "selectedcolumns", "sparkschema").forEach(key -> inputs.put(key, options.get(key)));
```

Comment:
Should we remove the word "schema" from here since we changed it to "sparkschema"?

Reply:
This is the TDE schema name.